embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/injection/DeliveryModule.kt

```diff
@@ -23,4 +24,5 @@ interface DeliveryModule {
     val cacheStorageService: PayloadStorageService?
     val requestExecutionService: RequestExecutionService?
     val schedulingService: SchedulingService?
+    val deliveryTracer: DeliveryTracer?
```

Comment:
Does it make sense for this to be part of `EmbLogger` since that's already being thread through everywhere, and we can add to this in the future by having more components contribute to this without also having to thread that through to all those other places?

Reply:
I'm not convinced it would be a good idea to add this onto the `EmbLogger` interface as it feels like this is a separate concern